### ISSUE TITLE
About the processing during test

### ISSUE THREAD

**Issue by hellojialee** [mks0601/PoseFix_RELEASE#7]
First, thank you for sharing your great work!
In Section 7.3 in your paper, there goes:

>  we used their released codes and pre-trained models. We tested them by ourselves without ensembling and testing time augmentation.

Did you mean using single input for all the related approaches (PAFs [5], AE [20], Mask R-CNN [11], CPN [7], and Simple [30])?  I'm not clear. Which procedure is yours in the paper:

1. Only single input to the model
2. Multi input to the model (probably with flip augmentation)
3. Multi input and flip augmentation but without model ensemble 

Because for bottom-up approaches, they must use multi-scale input images to cover the variant   scales of poses. And for other top-down approaches, they usually use flip augmentation to improve precision. 

Looking forward to your help. Thank you!

**Comment by mks0601** [comment]
Sorry for not clear explanation. I cannot remember exactly, however, I'm sure that I did not use testing time multi-scale augmentation and model ensemble. Maybe I used testing time flip but flip improved performance not that much (at most 1 mAP).

**Comment by hellojialee** [comment]
Thank you for your answer!
